lib/sorcery/controller.rb

```diff
@@ -63,6 +65,16 @@ def login(*credentials)
         end
       end
 
+      def login!(*credentials)
+        user = login(*credentials)
+
+        if user.nil?
+          raise Sorcery::InvalidCredentials
+        else
+          user
+        end
```

Comment:
```ruby
user = login(*credentials)

raise Sorcery::Error::InvalidCredentials if user.nil?

user
```

I suspect RuboCop will bark about the spacing between lines, but this will at least use a guard clause instead of an if/else.